ui/components/MesheryPatterns.js

```diff
@@ -1197,8 +1198,13 @@ function MesheryPatterns({
           open={modalOpen.open}
           handleClose={handleModalClose}
           submit={
-            { deploy : () => handleDeploy(modalOpen.pattern_file, modalOpen.name), unDeploy : () => handleUnDeploy(modalOpen.pattern_file, modalOpen.name) }
+            {
+              deploy : () => handleDeploy(modalOpen.pattern_file, modalOpen.name),  unDeploy : () => handleUnDeploy(modalOpen.pattern_file, modalOpen.name), verify : () => handleVerify(modalOpen.pattern_file, modalOpen.name)
+            }
           }
+          verify={() => {
```

Comment:
Why is this needed??
You have provided `verify` callback here https://github.com/meshery/meshery/pull/6957/files#diff-e1d76eb522ad8ca1078d26e59d9c2635f97eeeaefdd95f7c693683fe3e151bd5R1202.